redfish-core/lib/service_root.hpp

```diff
@@ -23,6 +23,105 @@
+                messages::internalError(asyncResp->res);
+                return;
+            }
+            // Iterate over all retrieved ObjectPaths.
+            for (const auto& object : subtree)
+            {
+                const std::string& path = object.first;
+                BMCWEB_LOG_DEBUG << "Got path: " << path;
+                const std::vector<
+                    std::pair<std::string, std::vector<std::string>>>&
+                    connectionNames = object.second;
+                if (connectionNames.empty())
+                {
+                    continue;
+                }
+
+                // This is not system, so check if it's cpu, dimm, UUID or
+                // BiosVer
+                for (const auto& connection : connectionNames)
+                {
+                    for (const auto& interfaceName : connection.second)
+                    {
+                        if (interfaceName ==
+                            "xyz.openbmc_project.Inventory.Item.System")
+                        {
+                            crow::connections::systemBus->async_method_call(
+                                [asyncResp](
+                                    const boost::system::error_code ec2,
+                                    const std::vector<
+                                        std::pair<std::string, VariantType>>&
+                                        propertiesList) {
+                                    if (ec2)
+                                    {
+                                        // doesn't have to include this
+                                        // interface
+                                        return;
+                                    }
+                                    BMCWEB_LOG_DEBUG
+                                        << "Got " << propertiesList.size()
+                                        << " properties for system";
+                                    for (const std::pair<std::string,
+                                                         VariantType>&
+                                             property : propertiesList)
+                                    {
+                                        const std::string& propertyName =
+                                            property.first;
+                                        if ((propertyName == "Model") ||
+                                            (propertyName == "SerialNumber"))
+                                        {
+                                            const std::string* value =
+                                                std::get_if<std::string>(
+                                                    &property.second);
+                                            if (value != nullptr)
+                                            {
+                                                asyncResp->res
+                                                    .jsonValue["ACF"]
```

Comment:
See comment below I think we should do  ["Oem"]["OpenBMC"][<propertyName>] if we do want ACF in the name ["Oem"]["OpenBMC"]["ACF"][<propertyName>]. I am also open to ["Oem"]["IBM"][<propertyName>] / ["Oem"]["IBM"]["ACF"][<propertyName>] but really think we want ["Oem"][<org>]like we have other places

Reply:
I went with ["Oem"]["IBM"].